From cc823367ab4624a34502121c5671c8a37fb6fbb6 Mon Sep 17 00:00:00 2001
From: Sashko <sashko506@gmail.com>
Date: Sun, 3 Nov 2019 02:36:57 +0200
Subject: [PATCH] sepolicy: Resolve nfc denial

 denied { search } for pid=236 comm=android.hardwar name=nfc dev=mmcblk0p29 ino=610801 scontext=u:r:hal_nfc_default:s0 tcontext=u:object_r:nfc_data_file:s0 tclass=dir permissive=0

Change-Id: I455e2ca9e35f287b757e958cbc1f67d19d903ac6
---
 sepolicy/hal_nfc_default.te | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 sepolicy/hal_nfc_default.te

diff --git a/sepolicy/hal_nfc_default.te b/sepolicy/hal_nfc_default.te
new file mode 100644
index 00000000..e183dd8e
--- /dev/null
+++ b/sepolicy/hal_nfc_default.te
@@ -0,0 +1 @@
+allow hal_nfc_default nfc_data_file:dir search;
